boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp.dts

```diff
@@ -105,6 +105,7 @@
 };
 
 &grtc {
+	child-owned-channels = <7 8 9 10 11>;
```

Comment:
Shouldn't this be SoC specific and not board specific?

Reply:
It was soc specific long time ago but on request of @gmarull it was moved to board DTS.

Reply:
this all depends if this can change from board to board or not. If not, then at soc dts is fine.

Reply:
then it should go to soc dts

Reply:
moved to `dts/arm/nordic/nrf54l15_cpuapp_peripherals.dtsi`